lib/utils/getFrameworkId.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const path = require('path');
+const getServerlessDir = require('./config/getServerlessDir');
+const readFileSync = require('./fs/readFileSync');
+const fileExistsSync = require('./fs/fileExistsSync');
+
+function getFrameworkId() {
```

Comment:
Would be nice if this can be moved to the `Utils` class in `lib/classes/Utils.js`.